src/clojure_lsp/crawler.clj

```diff
@@ -49,21 +49,24 @@
     (swap! db update :findings merge (group-by :filename (:findings result)))
     analysis))
 
+(defn ^:private analyze-test-paths! [paths db]
+  (map #(producer/refresh-test-tree % db) paths))
```

Comment:
I think we should pass a URI not a filename to this function, so probably we need to call `shared/filename->uri` before calling the producer.

Reply:
Hum, the paths are not necessarly filenames, right? they could be `/path/to/src/` and not `/path/to/src/my-file.clj`, and this would make the the test tree not work.
We should re-think a way to get all the URIs of the project, not only the paths.
Probably find it via clj-kondo analysis filtering project analysis only

Reply:
That's right, I was thinking if we had something like that in the project, but based on your comment it sounds like we don't have that feature at the moment, I will check if we can get that list of filenames from clj-kondo.

Thx Greg!

Reply:
@erickisos probably something like:
```clojure
(->> (:analysis @db)
       (map :filename)
       (filter #(string/starts-with "file:/"))) ;; filter project only filenames
```

Reply:
just in case, I tried with one similar approach, and checking around with something like this
```clojure
(:documents @db)
```

And it seems like at least kondo and clojure-lsp don't keep track of all the files from the project.

WDYT about adding that kind of function in clojure-lsp? Or could be better if I add that in clj-kondo? I mean, we can just keep that in the clojure-lsp crawler.

Reply:
the `:documents` are filled with the current opened documents on the editor and information about version and content, I think the best would be to use kondo analysis for that, which is more reliable, something like this should work :
```clojure
(-> (:analysis @db/db)
      (q/filter-project-analysis db/db)
      keys)
```